### ISSUE TITLE
Layout/IndentationWidth misses unindented blocks

### ISSUE THREAD

**Issue by saveriomiroddi** [rubocop-hq/rubocop#6360]
## Expected behavior

The `Layout/IndentationWidth` cop should apply to all the `else` blocks.

## Actual behavior

In some cases, the cop is not applied [to else blocks].

## Steps to reproduce the problem

See the following code:

```ruby
mycondition = rand >= 0.5

if mycondition
puts 'caught (if)'
else
puts 'caught (else)'
end

if mycondition
# comment, no code
else
puts 'not caught (else)'
end

if mycondition # nothing in the if block
else
puts 'not caught (else)'
end
```

## RuboCop version

```
$ rubocop -v
0.59.2
```

**Comment by bquorning** [comment]
I can reproduce this bug. Thank you for this report @saveriomiroddi.

**Comment by bquorning** [comment]
Proposed fix in #6548.
